### DISCUSSION TITLE
Java agent to monitor couchbase transactions

### DISCUSSION THREAD

**Discussion by shreyasbasavarajappa** [apache/skywalking#7973] [General]
Hi,
   We have Java agent deployed and it is monitoring mysql, elastic and other dbs. I want to know how to enable java agent to monitor couchbase transactions also. Is there any plugin available?

**Comment by wu-sheng** [comment]
No, there isn't. If you want to contribute plugin, we could help on review.

**Reply by shreyasbasavarajappa** [reply]
Could you help me with a starting point? Why couchbase wasn't picked in the first place and what were the challenges?

**Reply by wu-sheng** [reply]
Plugins for different frameworks and libs are contributed by different people globally. The simple reason would be no one contributed one.
There is no special technology reason, AFAIK.
